example-stm32f103c8/Cargo.toml

```diff
@@ -16,3 +16,4 @@ embedded-hal = "0.2.3"
 
 [dev-dependencies]
 cortex-m-rtfm = "0.5"
+stm32-usbd = "0.5"
```

Comment:
Why do you need this? `UsbBus` should be available as `stm32f1xx-hal::usb::UsbBus`.

Reply:
@Disasm Fixed, thanks for pointing it out!